src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2265,12 +2297,16 @@ public bool BackgroundImageSynchronosLoading
                 backgroundImageSynchronosLoading = value;
 
                 string bgUrl = null;
-                Background.Find(ImageVisualProperty.URL)?.Get(out bgUrl);
+                var pValue = Background.Find(ImageVisualProperty.URL);
+                pValue?.Get(out bgUrl);
+                pValue?.Dispose();
```

Comment:
It is ok, but I think it is better to check nullity once for better readability.
```
if (pValue != null)
{
    pValue.Get(out bgUrl);
    pValue.Dispose();
}
```

Reply:
> It is ok, but I think it is better to check nullity once for better readability.
> 
> ```
> if (pValue != null)
> {
>     pValue.Get(out bgUrl);
>     pValue.Dispose();
> }
> ```

OK. I will do it next time